pitop/robotics/drive_controller.py

```diff
@@ -28,30 +28,33 @@ def __init__(self, left_motor_port="M3", right_motor_port="M0", name="drive"):
         self.right_motor_port = right_motor_port
         self.left_motor_port = left_motor_port
 
-        self._wheel_separation = 0.1675
-        self._wheel_diameter = 0.074
+        self._wheel_separation = 0.163
+        self._wheel_diameter = 0.0718
         self._wheel_circumference = self._wheel_diameter * pi
         self._linear_speed_x_hold = 0
 
         self._left_motor_port = left_motor_port
         self._right_motor_port = right_motor_port
 
         self.left_motor = EncoderMotor(port_name=left_motor_port,
-                                       forward_direction=ForwardDirection.CLOCKWISE)
+                                       forward_direction=ForwardDirection.CLOCKWISE,
+                                       wheel_diameter=self._wheel_diameter)
```

Comment:
Not needed if a constant value is always passed in that matches the parameter's default value